db/db_test.cc

```diff
@@ -8810,7 +8825,7 @@ class RecoveryTestHelper {
     ASSERT_GT(fd, 0);
     ASSERT_EQ(offset, lseek(fd, offset, SEEK_SET));
 
-    char buf[len];
+    void* buf = alloca(len);
```

Comment:
@igorcanadi. Yes, this was in previous diff. This will not leak as this is a rough equivalent of reserving a stack space in a dynamic manner. For this reason, if valgrind does not complain, I'd advocate leaving this as is because it would be close to the original code perf wise.